### DISCUSSION TITLE
CoApp License Blocked

### DISCUSSION THREAD

**Discussion by radar54** [aclap-dev/video-downloadhelper#1528] [License & Purchase requests]
### What's your issue?

Hi,
I recently purchased a new PC and when I try to enter my **CoApp** license for Firefox, I keep getting a "License Blocked".  How do I fix this?
Thanks,
Ben

### The 10 first digits (only 10) of your license number or email address

365a49dff0

### I confirm that I did not not paste my full license number

- [x] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

When you bought your VDH license, you provided an E-mail address.  When this locked license thing happens, the VDH web site sends you an E-mail at that E-mail address.  If that E-mail address is no longer valid, then you'll have to wait for @jcfrog Jérôme, who does work for VDH, to help you.  He's in France so take the time zone difference into consideration.  But if you do get the E-mail, it contains a link you need to click to unlock your license.

**Reply by radar54** [reply]
Thanks, Willy

**Comment by jcfrog** [comment]
It seems you found our email, I checked your licenses, none is locked.

**Reply by radar54** [reply]
Thank you for checking
